src/Maestro/Maestro.MergePolicies/NoRequestedChangesMergePolicy.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.DarcLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Maestro.MergePolicies
+{
+    public class NoRequestedChangesMergePolicy : MergePolicy
+    {
+        public override string DisplayName => "No Requested Changes";
+
+        public override async Task EvaluateAsync(IMergePolicyEvaluationContext context, MergePolicyProperties properties)
+        {
+            IEnumerable<Review> reviews = await context.Darc.GetPullRequestReviewsAsync(context.PullRequestUrl);
+
+            // Look for any reviews that are "rejected" or "rejected"
```

Comment:
You said "rejected" twice
```suggestion
            // Look for any reviews that are "rejected"
```